llvm/lib/Target/AMDGPU/SIInstructions.td

```diff
@@ -2045,6 +2059,21 @@ def : GCNPat <
   (V_OR_B32_e64 (S_MOV_B32 (i32 0x00008000)), VGPR_32:$src) // Set sign bit
 >;
 
+def : GCNPat <
```

Comment:
These should be the same as the f16 case. Can you avoid repeating the patterns with a foreach vt : [f16, bf16] over the existing patterns

Reply:
Of course, done.